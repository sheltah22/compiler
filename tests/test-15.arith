let negbool : bool -> bool =
  fun (x: bool) : bool -> if x then false else true in
negbool 4
