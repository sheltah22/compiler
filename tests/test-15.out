Fatal error: exception Failure("Fun call typechecking failed, expected input type: bool, actual: int")
